packages/ui-extensions/docs/surfaces/customer-account/reference/apis/navigation.doc.ts

```diff
@@ -47,55 +29,7 @@ const data: ReferenceEntityTemplateSchema = {
       ],
     },
   },
-  // examples: {
-  //   description: 'Navigation api examples',
-  //   examples: [
-  //     {
-  //       codeblock: {
-  //         title: 'Listening for navigation current entry events',
-  //         tabs: [
-  //           {
-  //             code: '../examples/navigation/event-listeners.example.tsx',
-  //             language: 'jsx',
-  //             title: 'React',
-  //           },
-  //         ],
-  //       },
-  //     },
-  //     {
-  //       codeblock: {
-  //         title: 'Using the live current entry value in a full-page extension',
-  //         tabs: [
-  //           {
-  //             code: '../examples/navigation/use-current-entry.example.tsx',
-  //             language: 'jsx',
-  //             title: 'React',
-  //           },
-  //         ],
-  //       },
-  //     },
-  //     {
-  //       codeblock: {
-  //         title: 'Navigating to customer account native order index page',
-  //         tabs: [
-  //           {
-  //             code: '../examples/navigation/navigating-to-customer-account-page.example.tsx',
-  //             language: 'jsx',
-  //             title: 'React',
-  //           },
-  //         ],
-  //       },
-  //     },
-  //   ],
-  // },
-  related: [
-    {
-      name: 'StandardApi',
```

Comment:
There's no more standard api docs, this goes to 404 in production so I am just removing this from here